metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -264,8 +273,9 @@ def make_tableau_client(self, site: str) -> Server:
                 },
             )
 
-            # From https://stackoverflow.com/a/50159273/5004662.
-            server._session.trust_env = False
+            if not self.session_trust_env:
```

Comment:
This was something we discovered along the way. It seems to disable / bypass proxy settings which is not what we want in our case. That's why we made it configurable. Let me know what you think.